src/router/routes.js

```diff
@@ -1,10 +1,19 @@
-module.exports = [
+import experimentVersionFragment from '@/graphql/fragments/experimentVersion.graphql';
+
+const routes = [
 	{
 		path: '/',
 		name: 'homepage',
 		component: () => import('@/pages/Homepage/Homepage'),
 		meta: {
-			contentfulPage: () => 'home',
+			contentfulPage: (route, apollo) => {
```

Comment:
This is a real clever way to do the experiment at a route level. I think we could reuse this in the future for some other route level experiments

Reply:
It was a @emuvente Idea, so He is the mastermind behind this!